server/package.json

```diff
@@ -37,6 +37,7 @@
     "redux": "^3.3.1",
     "stylus": "~0.42.2",
     "tosource": "~0.1.1",
+    "unorm": "^1.4.1",
```

Comment:
I might be wrong but I think this is not needed anymore since node supports it by default now (Übersicht uses 5.1.1)